README.md

```diff
@@ -1,28 +1,133 @@
-# inmemory-pagination
+# In Memory-POSF
 
-## Filter & Search :
 
-Filtering is done by column name while search is performed across columns.
-Multiple values can be supplied at the same time.
+### Problem Statement
 
-`&` -> separates column names
-`|` -> separates multiple values
-`*` -> indicates search across all columns
+Struck with Datasource with no filtering,searching and pagination features?.
+UI is feeling Hard to handle bulk of data?.
+We provide you the best solution to overcome these issues with minimal overhead.
 
-### Examples:
+### Solution
 
-"name=bharath"
-"*=Vega|vegas"
-"multiMedia.[].name=CCCC&*=Vega|vegas&userId=4051"
+As we are developing so much software in today’s world and dealing with tons of data.
```

Comment:
Sounds like an incomplete sentence when you begin with `as`.

Reply:
ok